File: app/src/main/java/com/BlizzardArmory/warcraft/Equipment/Enchantment.java
package com.BlizzardArmory.warcraft.Equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Enchantment {

    @SerializedName("display_string")
    @Expose
    private String displayString;
    @SerializedName("enchantment_id")
    @Expose
    private int enchantmentId;
    @SerializedName("enchantment_slot")
    @Expose
    private EnchantmentSlot enchantmentSlot;

    public String getDisplayString() {
        return displayString;
    }

    public void setDisplayString(String displayString) {
        this.displayString = displayString;
    }

    public int getEnchantmentId() {
        return enchantmentId;
    }

    public void setEnchantmentId(int enchantmentId) {
        this.enchantmentId = enchantmentId;
    }

    public EnchantmentSlot getEnchantmentSlot() {
        return enchantmentSlot;
    }

    public void setEnchantmentSlot(EnchantmentSlot enchantmentSlot) {
        this.enchantmentSlot = enchantmentSlot;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Enchantment.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Enchantment.
 */
@Keep
data class Enchantment(

    @SerializedName("display_string")
    var displayString: String,

    @SerializedName("enchantment_id")
    var enchantmentId: Long,

    @SerializedName("enchantment_slot")
    var enchantmentSlot: EnchantmentSlot

)